src/components/common/Logo.jsx

```diff
@@ -0,0 +1,13 @@
+import { Link } from 'react-router-dom';
+import LogoImage from '@/assets/logo/logo.svg?url';
```

Comment:
로고나 아이콘 불러오실 때 [위키 글](https://github.com/Sprint-19-Rolling/Rolling-Frontend/wiki/03.-vscode-%EC%84%A4%EC%A0%95,-%ED%99%98%EA%B2%BD%EB%B3%80%EC%88%98-%EC%84%A4%EC%A0%95,-%EC%9D%B4%EB%AF%B8%EC%A7%80,-%EC%95%84%EC%9D%B4%EC%BD%98-%EC%97%90%EC%85%8B-%EC%82%AC%EC%9A%A9%EB%B2%95#3-%EB%A1%9C%EA%B3%A0-%EC%9D%B4%EB%AF%B8%EC%A7%80-%EC%95%84%EC%9D%B4%EC%BD%98-import) 참고하시면 좋을 것 같습니다!
logo.js와 icons.js에 import export 구문을 넣어서 관리하고 있어서 svg 컴포넌트로 바로 접근이 가능합니다!
혹시 svg 크기 변경하고 싶으신거면 바로 `width`, `height`값 props로 전달할 수 있습니다~~!

```suggestion
import logo from '@/assets/logo/logo';

const Logo = () => {
  return (
    <Link to='/'>
      <logo.Logo width={106} height={30} />
     </Link>
  )
}
```